frontend/src/components/hub/ContactAmbassadorButton.js

```diff
@@ -0,0 +1,69 @@
+import { Button } from "@material-ui/core";
+import { makeStyles } from "@material-ui/core/styles";
+import React, { useContext } from "react";
+import { redirect } from "../../../public/lib/apiOperations";
+import { startPrivateChat } from "../../../public/lib/messagingOperations";
+import Router from "next/router";
+import UserContext from "../context/UserContext";
+import getTexts from "../../../public/texts/texts";
+import Cookies from "universal-cookie";
+import ContactCreatorButtonInfo from "../communication/contactcreator/ContactCreatorButtonInfo";
+import SendIcon from "@material-ui/icons/Send";
+
+const useStyles = makeStyles(() => ({
+  root: {
+    zIndex: 1,
+    position: "fixed",
+    bottom: 0,
+    right: "200px",
+
+    "& > div": {
+      display: "none",
+    },
+
+    "&:hover": {
+      "& > div": {
+        display: "flex",
+      },
+    },
+  },
+}));
+
+export default function ContactAmbassadorButton({ localAmbassador }) {
+  const classes = useStyles();
+  const { locale, user } = useContext(UserContext);
+  const cookies = new Cookies();
+  const token = cookies.get("token");
+  const texts = getTexts({ page: "hub", locale: locale });
+
+  const handleClickContact = async (e) => {
+    e.preventDefault();
+
+    if (!user) {
+      return redirect("/signup", {
```

Comment:
```suggestion
      return redirect("`${getLocalePrefix(locale)}/signup`, {
```
Always use localization for links :) (Don't forget to import getLocalePrefix